Directory.Build.props

```diff
@@ -62,12 +62,11 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="Microsoft.CodeAnalysis" Version="2.10.0" />
-        <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="2.10.0" />
-        <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.9.10" />
-        <PackageReference Include="Microsoft.VisualStudio.Threading.Analyzers" Version="16.0.102" />
-        <PackageReference Include="StyleCop.Analyzers" Version="1.1.118" />
-        <PackageReference Include="Newtonsoft.Json" Version="12.0.1" />
```

Comment:
Do we not need Newtonsoft.Json anymore?

Reply:
Build and tests still work without it 🤷

Reply:
Well, at least locally. Looks like updating the analyzers has imposed new rules that must be corrected on the CI